althea_types/src/websockets/mod.rs

```diff
@@ -76,6 +78,85 @@ pub enum RouterWebsocketMessage {
     },
 }
 
+// just ignore the hardware info field on the timeseries data message for now
```

Comment:
was this required just because hardware_info doesn't implement partialeq? What issue did you run into trying to add that over in kernel interface?

Reply:
yes, exactly- and when adding partialeq to hardware info, a bunch more nested struct members needed partialeq impl as well and I wasn't sure we really needed all that. I can add it in if we think it's important, but the manual impl here was only added because I'm trying to use an assert_eq in the unit test. maybe instead I can just manually compare values in the unit test instead....

Reply:
chasing down all the nested structs with partialeq is probably worth the trouble, but this is a good solution, manually comparing values wouldn't be better because that's only useful in this test versus being useful in any future case where we may want to compare these structs.